hm_1/tasks/task_01_05.py

```diff
@@ -0,0 +1,21 @@
+"""
+Given a list of integers numbers "nums".
+You need to find a sub-array with length less equal to "k", with maximal sum.
+The written function should return the sum of this sub-array.
+Examples:
+    nums = [1, 3, -1, -3, 5, 3, 6, 7], k = 3
+    result = 16
+"""
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    sums = []
+    if not nums or k > len(nums):
```

Comment:
If k is 100, should we check the subarray of size 1?

Reply:
As I understood from the higher comment, yes we need to check it